### DISCUSSION TITLE
It would have been cool if you could maintain tags for milestone versions of the repo

### DISCUSSION THREAD

**Discussion by viktor-shcherb** [KellerJordan/modded-nanogpt#35] [General]
The table in README is nice, but for reproducibility of old results sake it would have been nice to have tags. Without them it is now impossible to roll back to earlier versions given that the repository is nearing 500 commits.

Appreciate the work you've done here!

**Comment by KellerJordan** [comment]
older versions can be conveniently recovered from the reproducible logs linked to each record

for example, the train_gpt2.py used by the 11/08 record is precisely the code contained in the following: https://github.com/KellerJordan/modded-nanogpt/blob/master/records/110824_CastBf16/a833bed8-2fa8-4cfe-af05-58c1cc48bc30.txt
